.travis.yml

```diff
@@ -2,12 +2,40 @@ language: cpp
 os: linux
 sudo: required
 services: docker
+
 branches:
   only:
-    - master
-    - devel
+  - master
+  - devel
+  # regex to match valid tag format or TravisCI won't trigger deploys when a new tag is pushed
+  - /^v[0-9]+\.[0-9]+\.[0-9]+.*$/
+
 script:
-  - make CONCURRENCY=4 package-debian-buster-dbuild
-  - make CONCURRENCY=4 package-debian-stretch-dbuild
-  - make CONCURRENCY=4 package-debian-jessie-dbuild
-  - make CONCURRENCY=4 package-fedora-22-dbuild
+- make CONCURRENCY=4 package-debian-buster-dbuild
+- make CONCURRENCY=4 package-debian-stretch-dbuild
+- make CONCURRENCY=4 package-debian-jessie-dbuild
+- make CONCURRENCY=4 package-fedora-22-dbuild
+
+before_deploy: |
+  # rename debian packages to be distribution specific
+  pushd packages
+  for DEBIAN_PKG in `ls debian-*/libsuperiormysqlpp-dev_*.deb`; do
```

Comment:
I recall `ls` not being recommended for noninteractive/scripting uses, mostly for output ambiguity, you can see the discussion on the topic for example here - https://unix.stackexchange.com/questions/128985/why-not-parse-ls-and-what-to-do-instead.
That said, we're working with finite directory tree with no weird cases, so may be fine.

Reply:
Good point, thanks.